File: Java/Array/Second Largest.java
//https://www.geeksforgeeks.org/problems/second-largest3735/1
class Solution {
    public int getSecondLargest(int[] a) {
        // Code Here
        int max=-1;
        for(int i= 0; i<a.length;i++){
            if(a[i]>max){
                max = a[i];
            }
        }
        int max2= -1;
        
        for(int i=0;i<a.length;i++){
            if(a[i]>max2 && a[i]!=max){
                max2=a[i];
            }
        }
       return max2;
    }
}

File: Python/Array/Second Largest.py
//https://www.geeksforgeeks.org/problems/second-largest3735/1
class Solution:
    def getSecondLargest(self, arr):
        # Code Here
        if len(arr) < 2:
            return -1  # or handle as needed
        
        max_val = -1
        for num in arr:
            if num > max_val:
                max_val = num

        max2 = -1
        for num in arr:
            if num != max_val and num > max2:
                max2 = num

        return max2
